Build of infographic broken

```
 :
npm ERR! Linux 3.10.0-862.2.3.el7.x86_64
npm ERR! argv "/opt/rh/rh-nodejs4/root/usr/bin/node" "/opt/rh/rh-nodejs4/root/usr/bin/npm" "install"
npm ERR! node v4.6.2
npm ERR! npm  v2.15.1
npm ERR! code EPEERINVALID
npm ERR! peerinvalid The package jsdom@7.0.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer mocha-jsdom@1.2.0 wants jsdom@>10.0.0
npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app-root/src/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -21,9 +21,9 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "esprima": "^2.7.2",
-    "jsdom": "7.0.2",
+    "jsdom": "11.11.0",
     "mocha": "^5.2.0",
-    "mocha-jsdom": "^1.1.0",
+    "mocha-jsdom": "^2.0.0",
     "proxyquire": "^1.7.9",
     "should": ">= 0.0.1"
   }

